test/integration_test.py

```diff
@@ -290,9 +290,15 @@ def __check_manifest(self, file: IO[bytes], uuid_field_name: str):
     def _download_with_drs(self, file_uuid: str):
         base_url = config.service_endpoint() + drs_http_object_path('')
         client = Client(base_url)
-        response = client.get_object(file_uuid)
-        self._validate_fastq_response_content(file_uuid, response.content)
-        log.info('Successfully downloaded file %s with DRS', file_uuid)
+        for access_method in ('https', 'gs'):
+            with self.subTest(access_method=access_method):
+                response = client.get_object(file_uuid, access_method=access_method)
+                if access_method == 'https':
+                    self._validate_fastq_response_content(file_uuid, response.content)
+                    log.info('Successfully downloaded file %s with DRS', file_uuid)
+                else:
```

Comment:
```suggestion
                elif access_method == 'gs':
```

and add `else: assert False`. Or unroll the loop, it's just one line shared.

Reply:
Suggestion added. I would unroll, but I like the idea of the subTest.